docs/_plugins/leaflet-providers.md

```diff
@@ -0,0 +1,12 @@
+---
+name: leaflet-providers
+category: basemap-providers
+repo: https://github.com/leaflet-extras/leaflet-providers
+author: leaflet-extras members
+author-url: https://github.com/leaflet-extras
+demo:
+compatible-v0: false
+compatible-v1: true
+---
+
+Contains configurations for various free tile providers & OSM, OpenCycleMap, Stamen, Esri, etc.
```

Comment:
Why are you adding the describtion without key? And not like `content: Contains configurations for va ....`?

Reply:
`content` is defined by the jekyll engine. See https://jekyllrb.com/docs/variables/